contrib/sarplus/python/pysarplus/SARPlus.py

```diff
@@ -488,3 +524,72 @@ def recommend_k_items(
             )
         else:
             raise ValueError("No cache_path specified")
+
+    def get_topk_most_similar_users(self, test, user, top_k=10):
```

Comment:
One question, have you checked that the results between the CPU version and this one are the same?

Reply:
Yes, we are using the same test cases as for the CPU version.